src/state/util/index.js

```diff
@@ -0,0 +1,64 @@
+export const asyncThunkUtils = slicePrefix => {
```

Comment:
lowkey one of the coolest things I've crafted for Redux... I love that our vanilla Redux is inching ever more closely to Redux Toolkit